java/app/src/main/java/com/google/android/gms/samples/wallet/activity/CheckoutActivity.java

```diff
@@ -105,18 +187,18 @@ private void initializeUi() {
     detailImage = findViewById(R.id.detailImage);
 
     findViewById(R.id.googlePayButton).setOnClickListener(
-            new View.OnClickListener() {
-              @Override
-              public void onClick(View view) {
-                requestPayment(view);
-              }
-            });
+        new View.OnClickListener() {
```

Comment:
Since you're modernizing the code, any chance you can use view binding here instead of findViewById?

Reply:
Ah, view binding, finally!
Added.